anomalib/models/cfa/config.yaml

```diff
@@ -66,49 +66,44 @@ optimization:
 
 # PL Trainer Args. Don't add extra parameter here.
 trainer:
-  accelerator: auto # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
-  accumulate_grad_batches: 1
-  amp_backend: native
-  auto_lr_find: false
-  auto_scale_batch_size: false
-  auto_select_gpus: false
-  benchmark: false
-  check_val_every_n_epoch: 1
+  enable_checkpointing: true
```

Comment:
Do these new parameters also work with 1.6.x or are they only for versions greater than 1.7.0

Reply:
Great question, haven't checked 🙃

Reply:
> Do these new parameters also work with 1.6.x or are they only for versions greater than 1.7.0

@ashwinvaidya17, the only parameter that PyTorch Lightning v1.6.* does not support was `inference_mode`. Since we do not use it anywhere in the code base yet, I've removed it from the configs and confirmed that the changes are backward-compatible.